packages/@aws-cdk/aws-amplify/package.json

```diff
@@ -109,6 +109,7 @@
     "node": ">= 10.12.0"
   },
   "stability": "experimental",
+  "maturity": "cfn-only",
```

Comment:
this module has some [constructs](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-amplify-readme.html) now